rpc-impl/src/main/java/org/sonarsource/sonarlint/core/rpc/impl/SonarLintRpcServerImpl.java

```diff
@@ -185,6 +192,11 @@ public NewCodeRpcService getNewCodeService() {
     return new NewCodeRpcServiceDelegate(this);
   }
 
+  @Override
+  public TaintVulnerabilityTrackingRpcService getTaintVulnerabilityService() {
```

Comment:
```suggestion
  public TaintVulnerabilityTrackingRpcService getTaintVulnerabilityTrackingService() {
```
for consistency with the type?